libc/src/stdlib/heap_sort.h

```diff
@@ -18,11 +18,12 @@ namespace internal {
 // A simple in-place heapsort implementation.
 // Follow the implementation in https://en.wikipedia.org/wiki/Heapsort.
 
-LIBC_INLINE void heap_sort(const Array &array) {
-  size_t end = array.size();
+template <typename A, typename F>
+void heap_sort(const A &array, const F &is_less) {
```

Comment:
For consistency, could you please mark all these functions as `LIBC_INLINE`?

Reply:
I'm surprised that template function should be marked with `inline`, AFAIK they can't run into ODR issues, or is this about giving the compiler an inlining hint?